doc/man/goto-instrument.1

```diff
@@ -1015,7 +1015,9 @@ force aggressive slicer to preserve all direct paths
 .SS "Code contracts:"
 .TP
 \fB\-\-apply\-loop\-contracts\fR
-check and use loop contracts when provided
+.TP
+\fB\-disable\-side\-effect\-check\fR
+UNSOUND OPTION. do not check the side-effect of loop contracts
```

Comment:
disable checking for the absence of side effects in loop contracts clauses

Reply:
Please add an explanation as to why anyone would want to use this option.